### PR TITLE
Clean up admonitions on power

### PR DESCRIPTION
**Pull request by tekstrand**
## What
1. Place esp32 note inline
2. Collapse calculator by default, place tip inside
3. Add headlines to all admonitions

## Why
I think it looks cleaner and makes the page easier to read. If we can agree on this new format, I'd like to spread this to all of our other pages as well. What do you all think?

### CODE REVIEW

File: docs/configuration/radio/power.mdx [hunk 1]
```diff
@@ -30,6 +27,10 @@ Automatically shut down a device after a defined time period if power is lost.
 
 ### ADC Multiplier Override
 
+:::info[ESP32 Only]
```
[the remark this answers is not in the document]
Comment (tekstrand) [reply docs/configuration/radio/power.mdx:30]: With brackets you can use markdown, other than that it doesn't matter

File: docs/configuration/radio/power.mdx [hunk 2]
```diff
@@ -14,10 +15,6 @@ Power settings are advanced configuration, most users should choose a role under
 
 The power config options are: Power Saving, Shutdown after losing power, ADC Multiplier Override, Wait Bluetooth Interval, Light Sleep Interval, Minimum Wake Interval, and Device Battery INA2xx Address. Power config uses an admin message sending a `Config.Power` protobuf.
```
[the remark this answers is not in the document]
Comment (tekstrand) [reply docs/configuration/radio/power.mdx:16]: Good call. I am having trouble thinking of the right words. Ideas?

[the remark this answers is not in the document]
Comment (tekstrand) [reply docs/configuration/radio/power.mdx:16]: We could always follow up with this change later, as this PR is technically a "clean up" PR vs improving content. Either way though we definitely should fix it up.

[the remark this answers is not in the document]
Comment (thebentern) [reply docs/configuration/radio/power.mdx:16]: Wifi, BLE, and Serial are turned off during sleep.

[the remark this answers is not in the document]
Comment (tekstrand) [reply docs/configuration/radio/power.mdx:16]: If you've turned sleep on, how would you turn it back off if those are disabled?

[the remark this answers is not in the document]
Comment (tekstrand) [reply docs/configuration/radio/power.mdx:16]: **THAT** feels noteworthy!

**Pull request merged**